app/screens/galoy-address-screen/galoy-address-screen.tsx

```diff
@@ -92,7 +92,10 @@ gql`
 export const GaloyAddressScreen = () => {
   const { LL } = useI18nContext()
   const isAuthed = useIsAuthed()
-  const { data } = useAddressScreenQuery({ fetchPolicy: "cache-first", skip: !isAuthed })
+  const { data } = useAddressScreenQuery({
+    fetchPolicy: "cache-and-network",
```

Comment:
If we refetch the query do we need this change?

Reply:
probably not a bad idea to avoid potential stale data

Reply:
Oops! It was the first option I tried. forgot to delete it. I've updated it now

Reply:
we don't need `cache-and-network` since we refetch the query